office365.json

```diff
@@ -4723,7 +4723,7 @@
         "wheel": [
             {
                 "module": "Django",
-                "input_file": "wheels/py3/Django-3.2.5-py3-none-any.whl"
```

Comment:
We get rid of 3.2.5 vulnerable version entirely here.

Reply:
It's previously kept b/c 4.0.2 only works on py3.8+ 

See https://pypi.org/project/Django/4.0.2/